tutornotes/plugin.py

```diff
@@ -53,6 +53,20 @@
     "{{ NOTES_DOCKER_IMAGE }}",
 ))
 
+@tutor_hooks.Filters.COMPOSE_MOUNTS.add()
+def _mount_edx_notes_api(volumes, name):
```

Comment:
Great feature! Could you please document it in the README?

Reply:
@regisb I've added a couple lines in the docs. Please let me know if any details are missing.